diff --git a/node_modules/react-native-actions-sheet/.bun-tag-f63306bfc7a5ca0f b/.bun-tag-f63306bfc7a5ca0f
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/src/utils.js b/dist/src/utils.js
index 97ed959e1a48614f1fc4eac086db480d72ba8c1d..20f37dbdaab5ac540c985011bc00691870cd5c0b 100644
--- a/dist/src/utils.js
+++ b/dist/src/utils.js
@@ -10,12 +10,11 @@ export function getDeviceHeight(statusBarTranslucent, height) {
     return height;
 }
 export var getElevation = function (elevation) {
+    if (Platform.OS === 'android') {
+        return { elevation: elevation };
+    }
     return {
-        elevation: elevation,
-        shadowColor: 'black',
-        shadowOffset: { width: 0.3 * elevation, height: 0.5 * elevation },
-        shadowOpacity: 0.2,
-        shadowRadius: 0.7 * elevation,
+        boxShadow: (0.3 * elevation) + "px " + (0.5 * elevation) + "px " + (0.7 * elevation) + "px rgba(0, 0, 0, 0.2)",
     };
 };
 export var SUPPORTED_ORIENTATIONS = [
